modules/oauth/src/main/resources/META-INF/mule-oauth2.xsd

```diff
@@ -92,6 +92,15 @@
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
+                <xsd:attribute name="externalRedirectionUrl" type="xsd:string" use="optional">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                            This is the redirection url to send to the user, and must match the one configured in the
+                            oauth provider. It won't create an endpoint by itself, and should be used when the
```

Comment:
What do you mean by "must match the one configured in the oauth provider"? Seems like you're referencing the AES module.